### DISCUSSION TITLE
Sample data for ImageAnalysis/video/

### DISCUSSION THREAD

**Discussion by parzival2** [UASLab/ImageAnalysis#26] [General]
Hi Mr. Olson,

I wanted to take a moment to express my gratitude and appreciation for the incredible work you have done in providing tools for everyone to access. 
I wanted to study more into how the conformal elements are drawn on HUD and the processes behind them. To execute the code one would need the data(Video and the flight data recorded) but I don't have the AuraUAS system to get the flight data. 
I am wondering if you can provide example data that I can run the `Video Analysis` on that would help me study and understand the code better. 
Thanking you

**Comment by clolsonus** [comment]
I do have some data I can share, but it's not in a convenient place to send
you a link and the video file is about 2Gb.  I will try to upload it to my
google drive and see if I can create a public link, but my home internet is
///slow/// and currently telling me an upload ETA of 17+ hours.

On Thu, Apr 20, 2023 at 6:04 AM Kalyan Sreenivas ***@***.***>
wrote:

> Hi Mr. Olson,
>
> I wanted to take a moment to express my gratitude and appreciation for the
> incredible work you have done in providing tools for everyone to access.
> I wanted to study more into how the conformal elements are drawn on HUD
> and the processes behind them. To execute the code one would need the
> data(Video and the flight data recorded) but I don't have the AuraUAS
> system to get the flight data.
> I am wondering if you can provide example data that I can run the Video
> Analysis on that would help me study and understand the code better.
> Thanking you
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/NorthStarUAS/ImageAnalysis/discussions/26>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACXYYDMY64WORLHYP25DTO3XCEJ5RANCNFSM6AAAAAAXFKUFVY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Curtis Olson

**Reply by parzival2** [reply]
Thank you for even considering to upload the data. Looking forward to run the script and understand the code 👍

**Reply by parzival2** [reply]
Hi Mr. Olson,
Just coming back to you to know whether you have been successful in uploading the sample data ?
Thank you :)

**Comment by parzival2** [comment]
Hi Mr. Olson,
Just coming back to you to know whether you have been successful in uploading the sample data ?
Thank you :)
